troubadix/plugins/script_family.py

```diff
@@ -87,6 +87,7 @@
     "Web application abuses",
     "Windows",
     "Windows : Microsoft Bulletins",
+    "Alma Linux Local Security Checks",
```

Comment:
Can you insert it, so that it's sorted correctly? You did the same thing for Rocky as well 👀

Reply:
Fixed